src/main/java/com/epam/esm/SpringConfig.java

```diff
@@ -0,0 +1,87 @@
+package com.epam.esm;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.support.ReloadableResourceBundleMessageSource;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+import org.springframework.web.servlet.LocaleResolver;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.springframework.web.servlet.i18n.CookieLocaleResolver;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+import javax.servlet.http.HttpServletRequest;
+import javax.sql.DataSource;
+import java.util.Locale;
+
+@SpringBootApplication
+@EnableWebMvc
+@EnableTransactionManagement
+@PropertySource("classpath:application.properties")
+public class SpringConfig implements WebMvcConfigurer {
+
+    private static final String ENCODING = "UTF-8";
+    private static final String SET_LOCALE = "classpath:locale/messages";
+    private static final String ACCEPTED_LANGUAGE = "Accept-language";
+    private static final String PERSISTENCE_UNIT_NAME = "app";
+
+    public static void main(String[] args) {
```

Comment:
This main method is usually located in a separate class

Reply:
Now the run() method moved to a separate MainEntryPoint class

Reply:
@a1exanddrovich I don't see `MainEntryPoint` class

Reply:
@a1exanddrovich now you have 2 `SpringConfig` classes. Please fix this.